docker/oso-host-monitoring/centos7/Dockerfile

```diff
@@ -76,6 +76,9 @@ RUN yum clean metadata && \
 ADD urllib3-connectionpool-patch /root/
 RUN yum install -y patch && yum clean all && cd /usr/lib/python2.7/site-packages/ && patch -p1 < /root/urllib3-connectionpool-patch
 
+RUN yum install -y python-pip
```

Comment:
@ivanhorvath No, pip is still not allowed.

This is a WIP (work in progress) PR, though, so @zgalor can do whatever he wants until it's ready to be reviewed.